tests/test_backend.py

```diff
@@ -702,6 +702,32 @@ def test_touch_default_timeout(self):
         self.assertEqual(self.cache.get("test_key"), "foo")
 
 
+class DjangoRedisCacheTestsWithClose(DjangoRedisCacheTests):
```

Comment:
This can be a decorator of the class

Reply:
I don't know how, could you share some documentation?

Reply:
Do not subclass `DjangoRedisCacheTests`, it will run tests twice, one for the parent class and one for the child